src/neptune/metadata_containers/metadata_container.py

```diff
@@ -703,3 +706,9 @@ def _fetch_entries(
     def get_root_object(self) -> "MetadataContainer":
         """Returns the same Neptune object."""
         return self
+
+    def log_traceback(self, stacktrace_lines: List[str]) -> None:
```

Comment:
I'm not a big fan of this approach. We moved the logic from background job to NeptuneObject but the background jobs should be rather opt-in and depends on the interface that NeptuneObject provides. Maybe we should prepare a way to trigger the handler in addition to `print_exception`